Resources/PinConfigs.kext/Contents/Info.plist

```diff
@@ -124,7 +124,7 @@
 					<key>Codec</key>
 					<string>agasecond - Realtek ALC256 (3246) for Xiaomi Pro Enhanced 2019</string>
 					<key>CodecID</key>
-					<integer>283902550</integer>
+					<int>283902550</int>
```

Comment:
Just wondering, shouldn't it be `integer` type rather than `int`? I know it shouldn't create any issues so far, but the other layouts use `integer`...